packages/schematics/migrations/legacy-migrations/20180515-switch-to-nx6.ts

```diff
@@ -34,13 +34,18 @@ export default {
         execSync('npm install', { stdio: [0, 1, 2] });
       }
 
-      execSync('ng update @angular/cli --from 1.7.4 --to 6.0.1', {
-        stdio: [0, 1, 2]
-      });
-      // TODO: remove --next before releasing 6.0.0 final
-      execSync('ng update @nrwl/schematics --from 1.0.3 --to 6.0.0 --next', {
-        stdio: [0, 1, 2]
-      });
+      execSync(
+        'ng update @angular/cli --from 1.7.4 --to 6.0.1 --migrate-only',
+        {
+          stdio: [0, 1, 2]
+        }
+      );
+      execSync(
+        'ng generate @schematics/update:migrate --package @nrwl/schematics --collection @nrwl/schematics/migrations/migrations.json --from 1.0.3 --to 6.0.0',
```

Comment:
The alternative to this is:
```ts
'ng update @nrwl/schematics --from 1.0.3 --to 6.0.0 --migrate-only'
```

But this isn't testable locally.
Speculating here but it might be because the version number is `0.0.1` and it thinks it should not go up to `6.0.0`?